gensrc/thrift/PlanNodes.thrift

```diff
@@ -573,17 +573,8 @@ struct TSortNode {
   // This is the number of rows to skip before returning results
   3: optional i64 offset
 
-  // TODO(lingbin): remove blew, because duplaicate with TSortInfo
```

Comment:
The upgrade process is generally a rolling upgrade. This means that there may be phases of a higher version of be and a lower version of fe. Are your changes compatible?